src/Presentation/Nop.Web.Framework/Infrastructure/Extensions/ApplicationPartManagerExtensions.cs

```diff
@@ -354,28 +354,9 @@ private static bool IsPluginDirectory(string directoryName)
         /// <param name="config"></param>
         private static void LoadPluginsInfo(NopConfig config)
         {
-            var useRedisToStorePluginsInfo = config.RedisEnabled && config.UseRedisToStorePluginsInfo;
-
-            //we use the main IRedisConnectionWrapper implementation since the DI isn't initialized yet
-            PluginsInfo = useRedisToStorePluginsInfo
-                ? new RedisPluginsInfo(_fileProvider, new RedisConnectionWrapper(config))
-                : new PluginsInfo(_fileProvider);
-
-            if (PluginsInfo.LoadPluginInfo() || useRedisToStorePluginsInfo || !config.RedisEnabled) 
-                return;
-
-            var redisPluginsInfo = new RedisPluginsInfo(_fileProvider, new RedisConnectionWrapper(config));
-
-            if (!redisPluginsInfo.LoadPluginInfo()) 
-                return;
+            PluginsInfo = new DbPluginsInfo(_fileProvider);
```

Comment:
using a new file storage method must be configurable

Reply:
Why would you not to always store the plug-in info in the DB? It makes the most since to be stored in the DB and I cannot see any benefit of storing outside of the DB over storing in the DB.